lib/foods.js

```diff
@@ -21,33 +21,41 @@ function checkInputs() {
     var calories = $('#calories-field input').val();
     var foodsRaw = sessionStorage.getItem("foods");
     var foods = $.parseJSON(foodsRaw);
-    foodTableSingleRow(food, calories, true, false, true)
-    saveFoodList();
+    foodTableSingleRow($('#food-list'), food, calories, true, false, true);
+    saveFoodList($('#food-list'));
     $('#name-field input').val("");
     $('#calories-field input').val("");
   };
 };
 
-function foodTableSingleRow(name, calories, deleteButton, checkBox, addToList){
+function foodTableSingleRow(whichTable, name, calories, deleteButton, checkBox, addToList){
   var oneRowHtml = '<tr class="food-row"><td class="food-name">' + name + '</td><td class="food-calories">' + calories + '</td>';
   if (deleteButton) {
     oneRowHtml += '<td class="food-delete"><button>&#10005;</button></td>';
   }
   oneRowHtml += '</tr>';
   if (addToList) {
-    $('tbody').prepend(oneRowHtml);
+    $(whichTable).prepend(oneRowHtml);
   } else {
-    $('tbody').append(oneRowHtml);
+    $(whichTable).append(oneRowHtml);    
   }
   createDeleteButton();
 };
 
+function storedTableName(whichTable){
+  if (whichTable.attr('id') == "food-list") {
+    return "foods";
+  } else if (whichTable.attr('id') == "meal-breakfast") {
+    return "mealBreakfast";
+  };
+};
+
 function loadFoodList(whichTable){
-  var foodsRaw = sessionStorage.getItem("foods");
+  var foodsRaw = sessionStorage.getItem(storedTableName(whichTable));
   var foods = $.parseJSON(foodsRaw);
```

Comment:
It should be. `sessionStorage` and `localStorage` are both instances of the `Storage` class, so they should all have the same methods.